# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/content/tests/chrome/test_window_intrinsic_size.xul
# Commit: b47a29dc541a
# Full Hash: b47a29dc541a8cefda7c14274581e9c868944400
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2019-08-15 09:46:49
# Regressor Bug: 1562292
# File Overlap Count: 1
# Description:
#   Bug 1562292: Part 2c - Use BrowserContext rather than DocShell to resolve named targets. r=nika
#   
#   This lets us lookup cross-process targets, but does not yet allow returning or
#   loading anything into them.
#   
# ==============================================================================

diff -r ce60426d7e31 -r b47a29dc541a toolkit/content/tests/chrome/test_window_intrinsic_size.xul
--- a/toolkit/content/tests/chrome/test_window_intrinsic_size.xul	Fri Jun 28 12:32:49 2019 -0700
+++ b/toolkit/content/tests/chrome/test_window_intrinsic_size.xul	Fri Jun 28 14:34:58 2019 -0700
@@ -11,7 +11,7 @@
 
   function openWindow(features = "") {
     return window.openDialog("window_intrinsic_size.xul",
-                             null, "chrome,dialog=no,all," + features);
+                             "", "chrome,dialog=no,all," + features);
   }
 
   function checkWindowSize(win, width, height, msg) {
